RadixWallet/Features/TransactionReviewFeature/TransactionReview.swift

```diff
@@ -729,7 +708,10 @@ extension TransactionReview {
 			}
 		}
 		do {
-			return try addingGuarantees(to: manifest, guarantees: state.allGuarantees)
+			return try Sargon.updatingManifestAddGuarantees(
```

Comment:
`mutatingManifest`? `changingManifest`? `modifyManifest`? or `updating` ? which is most clear?

Reply:
`modyManifest` sounds good